at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,621 @@
+      csleep
+      cexit
+      action_register
+    end
+  end
+
+  def action_forgot_password
+    clear
+    puts '========= Forgot Password ========='
+    print 'username : '
+    username = gets.chomp
+    if @session_controller.username_by_employee(username).count.zero?
+      puts 'Account does not exist !'
+      csleep
+      cexit
+      action_forgot_password
+    else
+      clear
+      print 'new password : '
+      password = gets.chomp
+      employee = Employee.new.tap do |e|
+        e.username = username
+        e.password = password
+      end
+      if @session_controller.forgot_password(employee)
+        puts 'Action forgot complete !'
+        csleep
+        action_login
+      else
+        puts 'Action forgot extis !'
+        csleep
+        cexit
+        action_forgot_password
+      end
+    end
+  end
+
+  def action_after_login
+    clear
+    puts '========= Task Manager ========='
+    puts '1. Team management'
+    puts '2. Leave management'
+    puts '3. Notification management'
+    puts '4. CSV'
+    print 'Action : '
+    number = begin
+               Integer(gets.chomp)
+             rescue StandardError
+               false
+             end
+
+    if number == false
+      puts 'input not string !'
+      csleep
+      cexit
+      action_after_login
+    end
+
+    action_team_manager if number == 1
+    action_leave_manager if number == 2
+    action_notification_manager if number == 3
+    action_csv_manager if number == 4
+  end
+
+  def action_team_manager
+    clear
+    account = @session_controller.current_user
+    query = account.to_a[0]
+    team_id = query['team_id']
+    role = query['role']
+
+    puts '========= Team Manager ========='
+    puts '0. Update a team' if role == 1
+    puts '1. Create a new team' if role == 1
+    puts '1. Add a new member' if role == 2
+    puts '2. Show list of members'
+    puts '3. Back'
+    puts '4. Logout'
+    puts '5. Exit program'
+    print 'Action : '
+    number = begin
+               Integer(gets.chomp)
+             rescue StandardError
+               false
+             end
+
+    if number == false
+      puts 'input not string !'
+      csleep
+      cexit
+      action_team_manager
+    end
+
+    if role == 1
+      new_team if number == 1
+      action_update_team if number.zero?
+    end
+
+    if role == 2
+      action_add_member_team if number == 1
+    end
+
+    action_show_members_team if number == 2
+    action_logout_back_exit number
+  end
+
+  def new_team
+    clear
+    puts '========= New Team ========='
+    print 'name team : '
+    name_team = gets.chomp
+    puts ''
+    print 'username leader : '
+    user_leader = gets.chomp
+    username = @session_controller.username_by_employee user_leader
+    user_id = username.to_a[0]['id']
+    role = username.to_a[0]['role']
+    if username.count.zero?
+      puts 'Username not extis !'
+      csleep
+      cexit
+      new_team
+    else
+      if role == 2
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7deb92b5e1c7275222ca44a289eebcc27cbc38fe/at-quannguyen/W4/main.rb#L223"></span>Style/IfInsideElse: Convert <code>if</code> nested inside <code>else</code> to <code>elsif</code>.</td>
    </tr>
  </tbody>
</table>